nameko/rpc.py

```diff
@@ -188,13 +188,17 @@ def send_response(self, result, exc_info):
         if exc_info is not None:
             error = serialize(exc_info[1])
 
+        # send response encoded the same way as was the request message
+        content_type = self.message.properties['content_type']
+        serializer = kombu.serialization.registry.type_to_name[content_type]
```

Comment:
Reinforcing the comment above: we should check earlier than here that there is a registered serializer for this `content_type`.

Reply:
True, it should be checked and it should raise ConfigurationError with a clear message rather than KeyErrors later on and on various places.

A check added.